lms/djangoapps/django_comment_client/management/management_utils.py

```diff
@@ -0,0 +1,49 @@
+'''
```

Comment:
Use double-quotes for docstrings, per the style guide [1].

[1] https://github.com/edx/edx-platform/wiki/Python-Guidelines